client/landing/gutenboarding/hooks/use-on-login.ts

```diff
@@ -27,9 +27,10 @@ export default function useOnSignup() {
 
 	const shouldTriggerCreate = useNewQueryParam();
 	const visibility = useNewSiteVisibility();
+	const isAnchorFmSignup = useSelect( ( select ) => select( ONBOARD_STORE ).getSelectedPodcastId() );
 
 	React.useEffect( () => {
-		if ( ! isCreatingSite && ! newSite && currentUser && shouldTriggerCreate ) {
+		if ( ! isCreatingSite && ! newSite && currentUser && shouldTriggerCreate && ! isAnchorFmSignup ) {
```

Comment:
`isAnchorFmSignup` is missing from `useEffect` deps.

Reply:
Filled out deps arrays here 30b0272d3c